packages/jsii-pacmak/lib/targets/go/types/class.ts

```diff
@@ -326,18 +327,18 @@ export class GoClass extends GoType {
   }
 }
 
-export class GoClassConstructor {
+export class GoClassConstructor extends GoMethod {
+  public readonly usesInitPackage = false;
+  public readonly usesRuntimePackage = false;
```

Comment:
Why `false`? The constructor actually does use `_init_` and `_jsii_`.

Reply:
Good catch. Fixing.

Reply:
I believe this was implicitly okay because these are enabled at the class level (for this reason).